.github/workflows/e2e-tests.yml

```diff
@@ -45,11 +43,19 @@ jobs:
       run: npm ci
     - name: Build applications
       run: |
-        ./gradlew :api-admin:build -x test
-        ./gradlew :api-admin:copyWebApp
-        ./gradlew :api-participant:build -x test
-        ./gradlew :api-participant:copyWebApp
+        REACT_APP_UNAUTHED_LOGIN=true ./gradlew :api-participant:jibDockerBuild -Djib.to.image=api-participant:ci
+        REACT_APP_UNAUTHED_LOGIN=true ./gradlew :api-admin:jibDockerBuild -Djib.to.image=api-admin:ci
```

Comment:
oooh, nice find on this `jib.to.image` opt.   How about we update the run_dockerized script to use this, and then just invoke that run_dockerized script here?  (and then we can get rid of that printVersion gradle task).